src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items = getNumbers(1, 42);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<string>('5');
```

Comment:
Using a string to represent the 'perPage' value can be problematic when performing arithmetic operations. It is better to store this value as a number to avoid the need for explicit conversions using the '+' operator.